lib/src/localization.dart

```diff
@@ -114,6 +119,9 @@ class Localization {
   }
 
   static PluralRule? _pluralRule(String? locale, num howMany) {
+    if (instance._ignorePluralRules) {
+      return () => _pluralCaseFallback(howMany);
```

Comment:
why not just

`return _pluralCaseFallback(howMany);`

Reply:
`PluralRule` is defined as function `typedef PluralRule = PluralCase Function();` so I need to return a function with no args.

![image](https://github.com/aissat/easy_localization/assets/4000539/d64ec019-53be-494f-9e6f-650bc3af26bb)